### ISSUE TITLE
[feature] implement selecting for mass edit

### ISSUE THREAD

**Issue by ColdIce1** [whyboris/Video-Hub-App#695]
Would like to propose a new feature: mass edit mode. You can drag / select to edit many files in one go. Such edits can be: delete, add tags, take screencaps etc...

**Comment by whyboris** [comment]
Thank you for the suggestion!

Currently you can already add a tag to many videos at once:
1. go to "Tags" in the bottom tray
2. click on "Batch mode" button on bottom left
3. click on as many videos as you'd like
4. click on any of the tags to add the tag to all the selected videos

Unfortunately this is not implemented for deleting files. 

Please elaborate on "take screencaps" -- what would that do?

**Comment by ColdIce1** [comment]
My proposal is similar to an edit mode in any other application, you click a pencil and you get the option to select files.
This edit mode can bring different features and I only suggested some like delete files & screencap feature.

The screencap feature lets you take screen captures of all the selected files at an interval and generate a screenshot of the file and save it in a picture, similar to: https://raw.githubusercontent.com/transitive-bullshit/ffmpeg-generate-video-preview/master/media/big-buck-bunny-6x5.jpg

**Comment by whyboris** [comment]
Thanks for the suggestion about screen captures. I don't think I'll add this feature to my application as it is complicated and there are so many options people would want to have to customize the "movie print". I strongly recommend a dedicated application for this purpose:

https://www.movieprint.org/

I hope you have seen there is already a "Full view" in the app which shows you all the screenshots for each file in the gallery 😉 I know it's not the same as exporting a `jpg` but it's close-enough 🤞

**Comment by ColdIce1** [comment]
Fair enough!  But would still love to see the edit mode to mass delete files and clean up directories with the already existing epic preview that I can't get away from...
